### DISCUSSION TITLE
Can I skip auditwheel in a manylinux build?

### DISCUSSION THREAD

**Discussion by FirefoxMetzger** [pypa/cibuildwheel#1212] [Q&A]
I have a ctypes-based project that opens a shared library at runtime to do its work. We used to distribute the python package and the shared library separately, but now I am working on building, bundling, and shipping them together. After adding the necessary plumbing, the wheel builds and tests fine on my windows and linux machines and now I am setting up cibuildwheels in GH actions to build and deploy the wheels.

Unfortunately, this doesn't work at the moment, because auditwheel tries to inspect the wheel after it has been built and fails with `INFO:auditwheel.main_repair:This does not look like a platform wheel`. This is because auditwheel seems to only search for extensions and - as far as I can tell - there is currently no way to make it pick up a shared library explicitly (https://github.com/pypa/auditwheel/issues/197).

Since this feature is missing from auditwheel, I'm wondering what my options are. Can I simply skip the auditwheel step? Is there a better alternative I am unaware off?

**Comment by Czaki** [comment]
Yes. You could set `repair-wheel-command` for Linux to empty value https://cibuildwheel.readthedocs.io/en/stable/options/#repair-wheel-command

**Reply by FirefoxMetzger** [reply]
That works.

At the same time, I'm a bit wary of ignoring this step entirely since - as far as I understand - the idea of auditwheel is to bundle any shared dependencies into the wheel. My package only depends on standard libraries (libgcc, libstdc++, libpthread), so I guess it should be fine, but it seems like a hard to discover/debug issue should it become one ... especially if the wheels of time made me forget that I am explicitly skipping this step 😅

**Reply by Czaki** [reply]
I do not have experience with such a scenario. Maybe someone else will help you.

**Reply by FirefoxMetzger** [reply]
No worries :) Thanks for pointing out the `CIBW_REPAIR_WHEEL_COMMAND` variable :)
